source/serverless.html.md

```diff
@@ -4,7 +4,7 @@ description: Kubeless Serverless plugin
 layout: page
 ---
 
-<img src="assets/images/serverless-logo.png">
+<img src="/assets/images/serverless-logo.png">
```

Comment:
Since we're using markdown and we don't need any special CSS class or style for the image, I'd rather use the markdown syntax:

```
![Alt text for the image. It's very important for accessibility](/assets/images/serverless-logo.png)
```